guides/v2.3/graphql/reference/configurable-product.md

```diff
@@ -3,18 +3,27 @@ group: graphql
 title: ConfigurableProduct endpoint
 ---
 
-The `ConfigurableProduct` endpoint defines which configurable product-specific attributes are returned when performing a `products` search.
+The `ConfigurableProduct` endpoint defines a query and mutation for configurable products.
```

Comment:
```suggestion
The `ConfigurableProduct` endpoint defines a mutation for adding configurable products to a cart. It also extends the `ProductInterface` so that attributes that are specific to configurable produ sects can be queried in a `products`
```